src/pages/git/git-pull/index.md

```diff
@@ -3,13 +3,31 @@ title: Git Pull
 ---
 ## Git Pull
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/git/git-pull/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+[**git pull**](https://git-scm.com/docs/git-pull) allows you to get new commits from a remote repository. To do this, *git pull* fetches ( *git fetch* ) the new commits and merges [(*git merge*)](https://guide.freecodecamp.org/git/git-merge) these into your local branch.
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+This command's syntax is as follows:
+```bash
+git pull [options] [<repository>]
+```
+in which:
+- **repository** is the URL to your repo. Example:  https://github.com/freeCodeCamp/freeCodeCamp.git
```

Comment:
For all bullets below, make the word all caps to match the command (it'll look a little odd, but should avoid people having any ambiguity about them). Also, switch the italics for *--quiet* and *--verbose* to use backticks 👍